util/csv-parser.js

```diff
@@ -0,0 +1,84 @@
+const {
+    Dish,
+    Tag,
+    User,
+    Restaurant,
+    Category,
+    Menu
+  } = require("../models");
+
+const { Op } = require("sequelize");
+
+
+const parse = require('csv-parse')
+const fs = require('fs');
+
+// converts list of allergen names to ids
+let allergensToIds = async (allergens) => {
+    allergenIds = []
+    allergens.split(",").forEach(async (allergen) => {
+        let tag = await Tag.findOne(
+            {where: { name: {[Op.iLike]: allergen.trim() }}}
+        )
+        if(tag) {
+            allergenIds.push(tag.id)
+        }
+    })
+
+    return allergenIds
+}
+
+let getOrCreateCategory = async (categoryName, menuId) => {
+    category = await Category.findOne({where: { name: categoryName, menuId: menuId }})
+    if(category) {
+        return category.id
+    } else {
+        newCategory = await Category.create({ name: categoryName, menuId: menuId })
+        return newCategory.id
+    }
+}
+
+let parseCSV = async (data, restaurantId, menuId, overwrite) => {
```

Comment:
What'll be the use of overwrite?

Reply:
it's not implemented yet because I'm not sure we need it, but it will allow the user to choose whether or not to overwrite a dish with the same name.  Right now it's enabled by default.